src/Codegen/Generator.hack

```diff
@@ -242,7 +242,11 @@ final class Generator {
         }
 
         foreach ($classish_objects as $class) {
-            if (\is_subclass_of($class->getName(), \Slack\GraphQL\Pagination\Connection::class)) {
+            $rc = new \ReflectionClass($class->getName());
+            if (
+                \is_subclass_of($class->getName(), \Slack\GraphQL\Pagination\Connection::class) &&
+                $rc->getAttributeClass(\Slack\GraphQL\ObjectType::class)
```

Comment:
Nice.

Should we get a test that this works? E.g., add a `FooConnection` abstract class subclassing `Connection`, and a `BarConnection` concrete class subclassing `FooConnection` and annotated with `ObjectType`?

Reply:
hmm yeah not really sure where/ how to add this to a test? I added a fixture with sublcassing, but what file do I test it in?

Reply:
Just testing that the codegen looks correct should be fine!

Reply:
hmm how do I do that? sorry not familiar with out this sort of testing works

Reply:
Basically:
* Add the appropriate connection classes to `tests/Fixtures`.
* Run tests using `make test`. This will do the codegen.
* Look at the generated files and confirm they look correct. E.g., if you declared an abstract `FooConnection` and a concrete `BarConnection`, we should end up with one generated class in a `BarConnection.hack`.
Also happy to pair!

Reply:
You also have failing tests... we'll need to annotate `UserConnection.hack` with the `ObjectType` attribute, and ideally look at that attribute in `Generator::getConnectionObjects`. Again, happy to pair.